addons/gut/plugin_control.gd

```diff
@@ -168,7 +172,7 @@ func _setup_gut():
 
 	_lgr = _utils.get_logger()
 	_gut = load('res://addons/gut/gut.gd').new()
-	_gut.connect('tests_finished', self, '_on_tests_finished')
+	_gut.connect('tests_finished',Callable(self,'_on_tests_finished'))
```

Comment:
This could use a `# !4.0 comment`.  Another example of connecting that might not work.